py/visdom/__init__.py

```diff
@@ -1491,7 +1504,8 @@ def scatter(self, X, Y=None, win=None, env=None, opts=None, update=None,
             if update == 'append':
                 if win is None:
                     update = None
-                elif not self.offline:
```

Comment:
Unintentional change?

Reply:
ya i guess some of indention error, I will resolve it